File_oM/Config/PushConfig.cs

```diff
@@ -40,6 +40,10 @@ public class PushConfig : ActionConfig
             "\nThis is the current standard for Datasets.")]
         public bool UseDatasetSerialization { get; set; } = false;
 
+        [Description("By default, certain types cannot be pushed to Json as root-level objects, for example numbers." +
```

Comment:
```suggestion
        [Description("By default, certain types cannot be pushed to Json as root-level objects, for example numbers.\n" +
```

Otherwise the two lines of string will be displayed to the user without any white space between them.